packages/content-conformance/src/runner.ts

```diff
@@ -51,7 +59,15 @@ export class ContentConformanceRunner {
   }
 
   async run() {
-    return this.engine?.execute()
+    if (this.status === RunnerStatus.RUNNING) return
+
+    this.status = RunnerStatus.RUNNING
+    try {
+      await this.engine?.execute()
+      this.status = RunnerStatus.SUCCESS
```

Comment:
@nayres What do you think about using `vfile-statistics` to determine success/failure after the run? Depending on the error count, we would set the status: https://www.npmjs.com/package/vfile-statistics

Reply:
I'll look into that! Probably a lot more useful for the user.